api/rs/slint/lib.rs

```diff
@@ -5,6 +5,9 @@
 
 #![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
 
+#[cfg(target_env = "musl")]
+compile_error!("Compiling with MUSL is not supported by this crate.");
```

Comment:
I think that if you move this to the bottom of file - or at least away from the module docs - then it will compile.